diff --git a/OpenKeychain/build.gradle b/OpenKeychain/build.gradle
index dee2bb0ad..ac054b886 100644
--- a/OpenKeychain/build.gradle
+++ b/OpenKeychain/build.gradle
@@ -177,7 +177,7 @@ android {
     defaultConfig {
         minSdkVersion 15
         targetSdkVersion 28
-        versionCode 53004
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName "5.3"
         applicationId "org.sufficientlysecure.keychain"
         // the androidjunitrunner is broken regarding coverage, see here:
