diff --git a/build.gradle b/build.gradle
index 6c9f24ba2..0b0997aa6 100644
--- a/build.gradle
+++ b/build.gradle
@@ -283,8 +283,8 @@ dependencyVerification {
     ]
 }
 
-def canonicalVersionCode = 495
-def canonicalVersionName = "4.42.0"
+def canonicalVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+def canonicalVersionName = new Date().format("yyyy.MM.dd-HH.mm")
 
 def postFixSize = 10
 def abiPostFix = ['armeabi-v7a' : 1,
@@ -304,8 +304,8 @@ android {
     }
 
     defaultConfig {
-        versionCode canonicalVersionCode * postFixSize
-        versionName canonicalVersionName
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
+        versionName new Date().format("yyyy.MM.dd-HH.mm")
 
         minSdkVersion 19
         targetSdkVersion 26
@@ -415,7 +415,7 @@ android {
 
             if (postFix >= postFixSize) throw new AssertionError("postFix is too large")
 
-            output.versionCodeOverride = canonicalVersionCode * postFixSize + postFix
+            output.versionCodeOverride = canonicalVersionCode
         }
     }
 
@@ -512,10 +512,6 @@ tasks.whenTaskAdded { task ->
     if (task.name.equals("assemblePlayRelease")) {
         task.finalizedBy signProductionPlayRelease
     }
-
-    if (task.name.equals("assembleWebsiteRelease")) {
-        task.finalizedBy signProductionWebsiteRelease
-    }
 }
 
 def getLastCommitTimestamp() {
