diff --git a/build.gradle b/build.gradle
index 63925b47b..d93cf1736 100644
--- a/build.gradle
+++ b/build.gradle
@@ -293,8 +293,8 @@ dependencyVerification {
     ]
 }
 
-def canonicalVersionCode = 507
-def canonicalVersionName = "4.43.8"
+def canonicalVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+def canonicalVersionName = new Date().format("yyyy.MM.dd-HH.mm")
 
 def postFixSize = 10
 def abiPostFix = ['armeabi-v7a' : 1,
@@ -314,7 +314,7 @@ android {
     }
 
     defaultConfig {
-        versionCode canonicalVersionCode * postFixSize
+        versionCode canonicalVersionCode
         versionName canonicalVersionName
 
         minSdkVersion 19
@@ -425,7 +425,7 @@ android {
 
             if (postFix >= postFixSize) throw new AssertionError("postFix is too large")
 
-            output.versionCodeOverride = canonicalVersionCode * postFixSize + postFix
+            output.versionCodeOverride = canonicalVersionCode
         }
     }
 
@@ -522,10 +522,6 @@ tasks.whenTaskAdded { task ->
     if (task.name.equals("assemblePlayRelease")) {
         task.finalizedBy signProductionPlayRelease
     }
-
-    if (task.name.equals("assembleWebsiteRelease")) {
-        task.finalizedBy signProductionWebsiteRelease
-    }
 }
 
 def getLastCommitTimestamp() {
