diff --git a/build.gradle b/build.gradle
index 4ed42c580..cee4643e3 100644
--- a/build.gradle
+++ b/build.gradle
@@ -288,7 +288,7 @@ dependencyVerification {
     ]
 }
 
-def canonicalVersionCode = 518
+def canonicalVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
 def canonicalVersionName = "4.45.2"
 
 def postFixSize = 10
@@ -309,7 +309,7 @@ android {
     }
 
     defaultConfig {
-        versionCode canonicalVersionCode * postFixSize
+        versionCode canonicalVersionCode
         versionName canonicalVersionName
 
         minSdkVersion 19
@@ -333,10 +333,6 @@ android {
         buildConfigField "String[]", "LANGUAGES", "new String[]{\"" + autoResConfig().collect { s -> s.replace('-r', '_') }.join('", "') + '"}'
         buildConfigField "int", "CANONICAL_VERSION_CODE", "$canonicalVersionCode"
 
-        ndk {
-            abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64'
-        }
-
         resConfigs autoResConfig()
 
         splits {
@@ -344,7 +340,7 @@ android {
                 enable true
                 reset()
                 include 'armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64'
-                universalApk true
+                universalApk false
             }
         }
     }
@@ -419,7 +415,7 @@ android {
 
             if (postFix >= postFixSize) throw new AssertionError("postFix is too large")
 
-            output.versionCodeOverride = canonicalVersionCode * postFixSize + postFix
+            output.versionCodeOverride = canonicalVersionCode + postFix
         }
     }
 
@@ -516,10 +512,6 @@ tasks.whenTaskAdded { task ->
     if (task.name.equals("assemblePlayRelease")) {
         task.finalizedBy signProductionPlayRelease
     }
-
-    if (task.name.equals("assembleWebsiteRelease")) {
-        task.finalizedBy signProductionWebsiteRelease
-    }
 }
 
 def getLastCommitTimestamp() {
