diff --git a/build.gradle b/build.gradle
index bac57a0c4..cf9115132 100644
--- a/build.gradle
+++ b/build.gradle
@@ -288,8 +288,8 @@ dependencyVerification {
     ]
 }
 
-def canonicalVersionCode = 512
-def canonicalVersionName = "4.44.4"
+def canonicalVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+def canonicalVersionName = new Date().format("yyyy.MM.dd-HH.mm")
 
 def postFixSize = 10
 def abiPostFix = ['armeabi-v7a' : 1,
@@ -309,7 +309,7 @@ android {
     }
 
     defaultConfig {
-        versionCode canonicalVersionCode * postFixSize
+        versionCode canonicalVersionCode
         versionName canonicalVersionName
 
         minSdkVersion 19
@@ -419,7 +419,7 @@ android {
 
             if (postFix >= postFixSize) throw new AssertionError("postFix is too large")
 
-            output.versionCodeOverride = canonicalVersionCode * postFixSize + postFix
+            output.versionCodeOverride = canonicalVersionCode
         }
     }
 
@@ -516,10 +516,6 @@ tasks.whenTaskAdded { task ->
     if (task.name.equals("assemblePlayRelease")) {
         task.finalizedBy signProductionPlayRelease
     }
-
-    if (task.name.equals("assembleWebsiteRelease")) {
-        task.finalizedBy signProductionWebsiteRelease
-    }
 }
 
 def getLastCommitTimestamp() {
