diff --git a/app/build.gradle b/app/build.gradle
index 4bcdd08..163357d 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -8,12 +8,12 @@ android {
         applicationId "org.proninyaroslav.libretorrent"
         minSdkVersion 15 /* Officially Android Jelly Bean (< 4.4) support is dropped in libtorrent4j */
         targetSdkVersion 28
-        versionCode 11
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         /*
          * Convection:
          *  major.minor.[revision][-some_info]
          */
-        versionName "1.9.1"
+        versionName new Date().format("yyyy.MM.dd-HH.mm")
         vectorDrawables.useSupportLibrary = true
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
 
@@ -29,7 +29,6 @@ android {
         }
         debug {
             applicationIdSuffix ".debug"
-            versionNameSuffix "-DEBUG"
 
             testCoverageEnabled true
             debuggable true
