diff --git a/app/build.gradle b/app/build.gradle
index 95a2912a..24977a54 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -19,7 +19,7 @@ android {
         applicationId 'fr.neamar.kiss'
         minSdkVersion 15
         targetSdkVersion 28
-        versionCode 158
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName "3.10.2"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
