diff --git a/app/build.gradle b/app/build.gradle
index d11c8e7..f8cb2b4 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -11,7 +11,7 @@ android {
         applicationId "com.jarsilio.android.scrambledeggsif"
         minSdkVersion 15
         targetSdkVersion 28
-        versionCode 33
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName "1.4.6"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         vectorDrawables.useSupportLibrary = true
