diff --git a/app/build.gradle b/app/build.gradle
index 4f0af8a..106e34f 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -30,7 +30,7 @@ android {
         applicationId "com.jarsilio.android.scrambledeggsif"
         minSdkVersion 15
         targetSdkVersion 28
-        versionCode 32
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName "1.4.5"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         vectorDrawables.useSupportLibrary = true
