diff --git a/build.gradle b/build.gradle
index 59f6a627d..3ec4de5e8 100644
--- a/build.gradle
+++ b/build.gradle
@@ -237,4 +237,12 @@ android {
         exclude 'META-INF/BCKEY.DSA'
         exclude 'META-INF/BCKEY.SF'
     }
+
+    def DateVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+    android.applicationVariants.all { variant ->
+        variant.outputs.all { output ->
+            output.versionCodeOverride = DateVersionCode
+        }
+    }
+
 }
