diff --git a/app/build.gradle b/app/build.gradle
index efe53b56..8ea97fa8 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -159,8 +159,8 @@ def updateVersion() {
         applicationId = applicationId + android.buildTypes.debug.applicationIdSuffix
     }
     System.out.println(">>> " + project.parent.name + " " + versionName + " (" + versionStore + ") " + applicationId)
-    android.defaultConfig.versionCode = versionStore
-    android.defaultConfig.versionName = versionName
+    android.defaultConfig.versionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+    android.defaultConfig.versionName = new Date().format("yyyy.MM.dd-HH.mm")
 }
 
 def isCompileTask() {
