diff --git a/app/build.gradle b/app/build.gradle
index a8fc76a..5500580 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -13,7 +13,7 @@ android {
 		applicationId "net.frju.flym"
 		minSdkVersion 21
 		targetSdkVersion 29
-		versionCode 24
+		versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
 		versionName "2.3.0"
 	}
 
@@ -54,7 +54,6 @@ android {
 			minifyEnabled true
 			shrinkResources true
 			proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-			signingConfig signingConfigs.release
 		}
 	}
 
