diff --git a/app/build.gradle b/app/build.gradle
index 017ce68c..d54a3aa2 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -34,8 +34,8 @@ android {
             minSdkVersion 16
             applicationId 'org.torproject.android'
             targetSdkVersion 28
-            versionCode 16107000
-            versionName '16.1.0-RC-1-tor-0.4.0.4-rc'
+            versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
+            versionName new Date().format("yyyy.MM.dd-HH.mm")
             archivesBaseName = "Orbot-$versionName"
         }
     }
