diff --git a/app/build.gradle b/app/build.gradle
index a3480f34..876d9393 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -11,7 +11,7 @@ android {
         applicationId "com.kunzisoft.keepass"
         minSdkVersion 14
         targetSdkVersion 27
-        versionCode = 22
+        versionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName = "2.5.0.0beta22"
         multiDexEnabled true
 
