diff --git a/app/build.gradle b/app/build.gradle
index ed0bb674..f69a9bd2 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -11,7 +11,7 @@ android {
         applicationId "com.kunzisoft.keepass"
         minSdkVersion 14
         targetSdkVersion 28
-        versionCode = 23
+        versionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName = "2.5.0.0beta23"
         multiDexEnabled true
 
