diff --git a/app/build.gradle b/app/build.gradle
index ad8f2c27..db2dc349 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -56,6 +56,12 @@ android {
     lintOptions {
         warning 'MissingTranslation', 'GetLocales', 'VectorDrawableCompat'
     }
+    def DateVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+    android.applicationVariants.all { variant ->
+        variant.outputs.all { output ->
+            output.versionCodeOverride = DateVersionCode
+        }
+    }
 }
 
 dependencies {
