diff --git a/app/build.gradle b/app/build.gradle
index 743882e6..37d5f0d9 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -55,6 +55,13 @@ android {
         checkReleaseBuilds false
         abortOnError false
     }
+
+    def DateVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+    android.applicationVariants.all { variant ->
+        variant.outputs.all { output ->
+            output.versionCodeOverride = DateVersionCode
+        }
+    }
 }
 
 configurations.all {
