diff --git a/app/build.gradle b/app/build.gradle
index 5d00d3ff..b250c4ff 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -28,7 +28,7 @@ android {
         applicationId "com.aurora.store"
         minSdkVersion 21
         targetSdkVersion 29
-        versionCode 9
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName "3.0.9"
 
         resConfigs "ar-rSA", "b+ast", "bh-rIN", "bg-rBG", "ca-rES", "de-rDE", "es-rES", "el-rGR", "fr-rFR", "gl-rES", "hi-rIN", "hr-rHR", "hu-rHU", "in-rID", "it-rIT", "iw-rIL", "ja-rJP", "ko-rKR", "lt-rLT",
