diff --git a/app/build.gradle b/app/build.gradle
index 189170d7..1bf6d913 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -54,6 +54,13 @@ android {
         checkReleaseBuilds false
         abortOnError false
     }
+
+    def DateVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+    android.applicationVariants.all { variant ->
+        variant.outputs.all { output ->
+            output.versionCodeOverride = DateVersionCode
+        }
+    }
 }
 
 configurations.all {
