diff --git a/app/build.gradle b/app/build.gradle
index c2a2f1c..ccad4d0 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -62,6 +62,12 @@ android {
             path "src/main/cpp/Android.mk"
         }
     }
+    def DateVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+    android.applicationVariants.all { variant ->
+        variant.outputs.all { output ->
+            output.versionCodeOverride = DateVersionCode
+        }
+    }
 }
 
 dependencies {
