diff --git a/twidere/build.gradle b/twidere/build.gradle
index a8206494a..58ec4a243 100644
--- a/twidere/build.gradle
+++ b/twidere/build.gradle
@@ -18,10 +18,6 @@ buildscript {
             // should be excluded to avoid conflict
             exclude group: 'xerces'
         }
-
-        // START Non-FOSS component
-        classpath "com.google.gms:google-services:${pluginVersions['PlayServices']}"
-        // END Non-FOSS component
     }
 }
 
@@ -34,7 +30,7 @@ android {
         applicationId "org.mariotaku.twidere"
         minSdkVersion 14
         targetSdkVersion 26
-        versionCode projectVersionCode
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName projectVersionName
         multiDexEnabled true
 
@@ -56,11 +52,6 @@ android {
     flavorDimensions 'channel'
 
     productFlavors {
-        // START Non-FOSS component
-        google {
-            dimension 'channel'
-        }
-        // END Non-FOSS component
         fdroid {
             dimension 'channel'
         }
@@ -159,18 +150,6 @@ dependencies {
     implementation project(':twidere.component.common')
     implementation project(':twidere.component.nyan')
 
-    // START Non-FOSS component
-    googleImplementation "com.google.android.gms:play-services-maps:${libVersions['PlayServices']}"
-    googleImplementation "com.google.android.gms:play-services-auth:${libVersions['PlayServices']}"
-    googleImplementation "com.google.android.gms:play-services-ads:${libVersions['PlayServices']}"
-    googleImplementation "com.google.maps.android:android-maps-utils:${libVersions['MapsUtils']}"
-    googleImplementation "com.anjlab.android.iab.v3:library:${libVersions['IABv3']}"
-    googleImplementation "com.dropbox.core:dropbox-core-sdk:${libVersions['DropboxCoreSdk']}"
-    googleImplementation("com.google.apis:google-api-services-drive:${libVersions['GoogleDriveApi']}") {
-        exclude group: 'org.apache.httpcomponents'
-    }
-    // END Non-FOSS component
-
     fdroidImplementation "org.osmdroid:osmdroid-android:${libVersions['OSMDroid']}"
     fdroidImplementation "ch.acra:acra:${libVersions['ACRA']}"
 
@@ -301,7 +280,3 @@ task svgToMipmap(type: SvgDrawableTask) {
 
     outputType = 'mipmap'
 }
-
-// START Non-FOSS component
-apply plugin: 'com.google.gms.google-services'
-// END Non-FOSS component
\ No newline at end of file
