diff --git a/app/build.gradle b/app/build.gradle
index e21c2da..8aba479 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -26,6 +26,12 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    def DateVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+    android.applicationVariants.all { variant ->
+        variant.outputs.all { output ->
+            output.versionCodeOverride = DateVersionCode
+        }
+    }
 }
 
 dependencies {
diff --git a/app/src/main/assets/env b/app/src/main/assets/env
index a1039cb..339f141 160000
--- a/app/src/main/assets/env
+++ b/app/src/main/assets/env
@@ -1 +1 @@
-Subproject commit a1039cb5e1732e0b8d7c602636a14bbd31a1ae7c
+Subproject commit 339f1415571ee5c835db8e36ede2d1d10fa8f321-dirty
