diff --git a/app/build.gradle b/app/build.gradle
index 7930021..3a288a8 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -47,7 +47,7 @@ android {
         minSdkVersion 21
         targetSdkVersion 28
         versionName androidGitVersion.name()
-        versionCode androidGitVersion.code()
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         ndk {
             abiFilters 'arm64-v8a', 'armeabi-v7a'
@@ -134,14 +134,6 @@ android {
         beta.res.srcDirs += 'src/build-official/res'
         beta.java.srcDirs += 'src/build-official-imports/kotlin'
     }
-    signingConfigs {
-        release {
-            keyAlias 'blokada'
-            keyPassword BLOKADA_KEY_PASSWORD
-            storeFile file(BLOKADA_KEY_PATH)
-            storePassword BLOKADA_STORE_PASSWORD
-        }
-    }
     buildTypes {
         debug {
         }
@@ -149,7 +141,6 @@ android {
             minifyEnabled false
             useProguard false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-            signingConfig signingConfigs.release
         }
         official {
             initWith(release)
