diff --git a/app/build.gradle b/app/build.gradle
index 141e7ec6..1e595aea 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -38,7 +38,7 @@ android {
         minSdkVersion 16
         targetSdkVersion 27
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-        versionCode 41
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName "0.8.4"
 
         buildConfigField "String", "COMMIT_COUNT", "\"${getCommitCount()}\""
