diff --git a/app/k9mail/build.gradle b/app/k9mail/build.gradle
index 0b62efed4..7f93d895a 100644
--- a/app/k9mail/build.gradle
+++ b/app/k9mail/build.gradle
@@ -39,7 +39,7 @@ android {
         applicationId "com.fsck.k9"
         testApplicationId "com.fsck.k9.tests"
 
-        versionCode 27000
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName '5.700-SNAPSHOT'
 
         minSdkVersion buildConfig.minSdk
diff --git a/gradle.properties b/gradle.properties
index a01ef3e45..9660de210 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -1,3 +1,4 @@
+org.gradle.jvmargs=-Xmx4096m
 android.useAndroidX=true
 android.enableJetifier=true
 org.gradle.parallel=true
