diff --git a/app/build.gradle b/app/build.gradle
index ac69c78..15e329e 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -38,6 +38,12 @@ android {
             dimension = 'market'
         }
     }
+    def DateVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+    android.applicationVariants.all { variant ->
+        variant.outputs.all { output ->
+            output.versionCodeOverride = DateVersionCode
+        }
+    }
 }
 
 dependencies {
