diff --git a/app/build.gradle b/app/build.gradle
index 0596ad1..37cf8cd 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -29,6 +29,13 @@ android {
     dataBinding {
         enabled = true
     }
+
+    def DateVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+    android.applicationVariants.all { variant ->
+        variant.outputs.all { output ->
+            output.versionCodeOverride = DateVersionCode
+        }
+    }
 }
 
 dependencies {
