diff --git a/app-android/app/build.gradle b/app-android/app/build.gradle
index 0e5c4e39..5ffce1c8 100644
--- a/app-android/app/build.gradle
+++ b/app-android/app/build.gradle
@@ -7,7 +7,7 @@ android {
         applicationId "de.tutao.tutanota"
         minSdkVersion 21
         targetSdkVersion 26
-        versionCode 359050
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName "3.59.5"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
