diff --git a/app/build.gradle b/app/build.gradle
index 9d40aa4..586eb30 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -17,13 +17,13 @@ android {
 		minSdkVersion 16
 		targetSdkVersion 26
 		versionName '1.15.0'
-		versionCode 50
+		versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
 	}
 
 	splits {
 		abi {
 			enable true
-			universalApk true
+			universalApk false
 		}
 	}
 
diff --git a/build.gradle b/build.gradle
index 99228c3..c1c9b29 100644
--- a/build.gradle
+++ b/build.gradle
@@ -1,7 +1,7 @@
 buildscript {
 	repositories {
-		jcenter()
 		google()
+		jcenter()
 	}
 	dependencies {
 		classpath 'com.android.tools.build:gradle:3.1.0'
@@ -13,8 +13,8 @@ allprojects {
 		if (project.hasProperty('MAVEN_REPO')) {
 			maven { url MAVEN_REPO }
 		}
-		jcenter()
 		google()
+		jcenter()
 		maven { url 'http://maven.ghostscript.com/' }
 	}
 }
