diff --git a/app/build.gradle b/app/build.gradle
index c2562fe9..2dfc9090 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -26,8 +26,8 @@ android {
     defaultConfig {
         minSdkVersion 19
         targetSdkVersion 28
-        versionCode 46
-        versionName "3.2"
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
+        versionName new Date().format("yyyy.MM.dd-HH.mm")
     }
 
     buildTypes {
@@ -82,4 +82,4 @@ dependencies {
 
     // Only compile Firebase ads for the free flavor.
     freeImplementation 'com.google.firebase:firebase-ads:18.1.0'
-}
\ No newline at end of file
+}
