diff --git a/app/build.gradle b/app/build.gradle
index 15b6d700..03b5453a 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -62,6 +62,14 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
+    def DateVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+    android.applicationVariants.all { variant ->
+        variant.outputs.all { output ->
+            output.versionCodeOverride = DateVersionCode
+        }
+    }
+
 }
 
 dependencies {
@@ -82,4 +90,4 @@ dependencies {
 
     // Only compile Firebase ads for the free flavor.
     freeImplementation 'com.google.firebase:firebase-ads:18.2.0'
-}
\ No newline at end of file
+}
