diff --git a/TMessagesProj/build.gradle b/TMessagesProj/build.gradle
index 627454c0..8e44884e 100644
--- a/TMessagesProj/build.gradle
+++ b/TMessagesProj/build.gradle
@@ -261,7 +261,8 @@ android {
     defaultConfig {
         minSdkVersion 16
         targetSdkVersion 27
-        versionName "5.6.1"
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
+		versionName new Date().format("yyyy.MM.dd-HH.mm")
 
         vectorDrawables.generatedDensities = ['mdpi', 'hdpi', 'xhdpi', 'xxhdpi']
         //obtain your own keys at https://core.telegram.org/api/obtaining_api_id
diff --git a/TMessagesProj/jni/boringssl b/TMessagesProj/jni/boringssl
--- a/TMessagesProj/jni/boringssl
+++ b/TMessagesProj/jni/boringssl
@@ -1 +1 @@
-Subproject commit 79ab5e8faa04871e13542e35374657185adee5c8
+Subproject commit 79ab5e8faa04871e13542e35374657185adee5c8-dirty
diff --git a/TMessagesProj/jni/ffmpeg b/TMessagesProj/jni/ffmpeg
--- a/TMessagesProj/jni/ffmpeg
+++ b/TMessagesProj/jni/ffmpeg
@@ -1 +1 @@
-Subproject commit 4154f8967820ca734a77ce91bb590cd77649dee8
+Subproject commit 4154f8967820ca734a77ce91bb590cd77649dee8-dirty
diff --git a/TMessagesProj/jni/libtgvoip b/TMessagesProj/jni/libtgvoip
--- a/TMessagesProj/jni/libtgvoip
+++ b/TMessagesProj/jni/libtgvoip
@@ -1 +1 @@
-Subproject commit 78decc81bf25cf36ad1b4a9398aa11cb195db9c5
+Subproject commit 78decc81bf25cf36ad1b4a9398aa11cb195db9c5-dirty
