diff --git a/app/build.gradle b/app/build.gradle
index cb57a31b..bd260a03 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -8,7 +8,7 @@ android {
         applicationId "org.schabi.newpipe"
         minSdkVersion 19
         targetSdkVersion 28
-        versionCode 740
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName "0.16.2"
 
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
