diff --git a/app/build.gradle b/app/build.gradle
index 12f1ac5f..aa5d1503 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -40,6 +40,13 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
+    def DateVersionCode = Integer.parseInt(new Date().format("yyMMddHHmm"))
+    android.applicationVariants.all { variant ->
+        variant.outputs.all { output ->
+            output.versionCodeOverride = DateVersionCode
+        }
+    }
 }
 
 ext {
