diff --git a/app/build.gradle b/app/build.gradle
index e730d10ed..42d207d47 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -25,7 +25,7 @@ android {
     buildToolsVersion '27.0.3'
 
     defaultConfig {
-        versionCode 1007050
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName getVersionName()
 
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
