diff --git a/app/build.gradle b/app/build.gradle
index 1974f0c..b747c53 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -9,17 +9,9 @@ android {
         applicationId "com.github.axet.audiorecorder"
         minSdkVersion 9
         targetSdkVersion 26
-        versionCode 331
+        versionCode Integer.parseInt(new Date().format("yyMMddHHmm"))
         versionName "3.3.6"
     }
-    signingConfigs {
-        release {
-            storeFile file(prop('RELEASE_STORE_FILE')?:'none')
-            storePassword prop('RELEASE_STORE_PASSWORD')
-            keyAlias prop('RELEASE_KEY_ALIAS')
-            keyPassword prop('RELEASE_KEY_PASSWORD')
-        }
-    }
     buildTypes {
         debug {
             debuggable true
@@ -30,7 +22,6 @@ android {
             shrinkResources true
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-            signingConfig signingConfigs.release
         }
     }
     applicationVariants.all { variant ->
